### DISCUSSION TITLE
How do i configure typescript path mapping?

### DISCUSSION THREAD

**Discussion by vladcosorg** [vitest-dev/vitest#1246] [General]
```
{
  "compilerOptions": {
    "paths": {
      "@/*": ["../*"]
    }
  },
```

When running the tests, vitest does not find any such paths. Jest had moduleNameMapper, how do i do same thing here?]

**Comment by sheremet-va** [comment]
You can use Vite's [alias feature](https://vitejs.dev/config/#resolve-alias)
```js
// vitest.config.js

export default {
  resolve: {
    alias: {
       '@' : path.resolve(__dirname, './src')
    }
  }
}
```

**Reply by vladcosorg** [reply]
Thanks,  didn't know that Vite options are available for use in Vitest. Is there more info about that on docs, did I miss it?
